<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditMealStuDialog</class>
 <widget class="QDialog" name="EditMealStuDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,3,2,2,1,0,1">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>列表（学号）：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnAdd">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnRemove">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>移除</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>预览：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>289</width>
        <height>105</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelPreview">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditMealStuDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAdd</sender>
   <signal>clicked()</signal>
   <receiver>EditMealStuDialog</receiver>
   <slot>addMealStu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>1018</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRemove</sender>
   <signal>clicked()</signal>
   <receiver>EditMealStuDialog</receiver>
   <slot>removeMealStu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>712</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addMealStu()</slot>
  <slot>removeMealStu()</slot>
 </slots>
</ui>
